Users/Install.rst

```diff
@@ -83,10 +83,77 @@ To install coala only (without any bears), you can do:
     **Ubuntu Users** - Do not use ``apt-get install coala``
     for installing coala as that is different software.
 
+Installing inside a pipenv
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pipenv is  user friendly method of managing virtual environments and
+packages.
+You can look at it as a mix of ``pip`` and ``virtualenv``. So you don't have
+to manage them seperately for your projects.
+
+First, assuming you have installed python and pip earlier from
+this guide we'll now install ``pipenv`` by running the following command.
+
+::
+
+    $ pip install pipenv
+
+Now we create a python 3 environment for our project, move to your project
+directory and run the following command.
+
+::
+
+    $ pipenv --three
+
+The above command will create virtual environment isolated from your system
+global installation.
+
+.. note::
+
+    **Unix based** (OS X and Linux) - you can find the user base binary
+    directory for your pipenv by running ``python -m site --user-base``.
+
+    The result of the above command will be something similar to
+    ``/home/<Username>/.local``.
+
+    **Windows** - run the following command ``py -m site --user-site``
+
+    Replace the occurrence of ``site-packages`` in the result of the command
+    above with Scripts to get something like
```

Comment:
is `Scripts` referring to a folder? In that case I would enclose in double backticks so it gets rendered as code and people understand it's referring to somethin "code"-like :+1: